server/routes/sampledzimages.ts

```diff
@@ -0,0 +1,50 @@
+import fs from 'fs'
+import path from 'path'
+import serverconfig from '#src/serverconfig.js'
+
+/*
+given a sample, return it's singlecell data from dataset
+*/
+
+export const api: any = {
+	endpoint: 'sampledzimages',
```

Comment:
what's diff of this to the other `dzimages` route?

Reply:
This route loads all dzi images paths for specified sample_id and dataset. The other route loads the tiles based on the path provided by this endpoint.

Reply:
then please add this comment to top and replace the borrowed one

Reply:
Changed the comment.